File: Python Solutions/485. Max Consecutive Ones.py
def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
    current = 0; highest = 0
    for x in nums:
        if x == 1: current += 1
        else: 
            if highest < current: highest = current
            current = 0    
    if highest < current: highest = current
    return highest

File: C++ Solutions/485. Max Consecutive Ones.cpp
int findMaxConsecutiveOnes(vector<int>& nums) {
    int count = 0;
    int highest = 0;
        
    for (int i = 0; i < nums.size(); i++){
        if (nums[i] == 1) {count++;}
        else {
            if (count > highest) {highest = count;}
            count = 0;
        }
    }
    
    if (count > highest) {highest = count;}
    return highest;
}